packages/graphql/src/translate/where/create-where-and-params.ts

```diff
@@ -227,16 +218,29 @@ function createWhereAndParams({
                     whereInput: entry[1] as any,
                     context,
                     node: refNode,
-                    nodeVariable: `${collectedMap}.node`,
+                    nodeVariable: thisParam,
                     relationship,
-                    relationshipVariable: `${collectedMap}.relationship`,
+                    relationshipVariable,
                     parameterPrefix,
                     listPredicates: [listPredicate],
                 });
 
-                resultStr += connectionWhere[0];
-                resultStr += ")"; // close ALL
-                res.clauses.push(resultStr);
+                if (connectionWhere[0]) {
+                    const clause = listPredicateToSizeFunction(listPredicate, matchPattern, connectionWhere[0]);
+                    res.clauses.push(clause);
+                    res.params = { ...res.params, ...connectionWhere[1] };
+                }
+
+                // TODO: Change to EXISTS when filtering no longer in projection
```

Comment:
**question**
Is it worth keeping this commented code?

Reply:
My mistake!